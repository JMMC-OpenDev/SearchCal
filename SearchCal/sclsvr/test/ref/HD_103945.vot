version https://git-lfs.github.com/spec/v1
oid sha256:e7edda7792ee9a956f26aa13a12b0273ca5769fe101a7c7a6e8abd25682726f4
size 268643
