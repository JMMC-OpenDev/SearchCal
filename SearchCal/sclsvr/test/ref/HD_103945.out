version https://git-lfs.github.com/spec/v1
oid sha256:8821fb6320d77cefef462c4beb930b34bf3dfb6dfffa56636d20c0f5fa0b683a
size 420626
